src/docs/implementations/graphite.adoc

```diff
@@ -42,6 +42,13 @@ management.metrics.export.graphite:
     step: 1m
 ----
 
+== Graphite Tag Support
+
+As of Micrometer version 1.4.0, Micrometer supports exporting Graphite metrics using tags instead of the traditional hierarchical format, and exports metrics using the tag format by default.
```

Comment:
```suggestion
As of Micrometer version 1.4.0, Micrometer supports exporting Graphite metrics using tags instead of the traditional hierarchical format.  Metrics are exported by using the tag format by default unless any  `tagsAsPrefix` values are configured.
```

Not sure what we should say for versions... Support was added in 1.4.0 with the default set to always be true, in 1.5.0 it will be based on the `tagsAsPrefix` due to https://github.com/micrometer-metrics/micrometer/pull/2007

Reply:
I'd prefer to avoid "in version X this, in version Y that" type things in the documentation. Let's wait until 1.5 is released and then merge this with the up-to-date behavior. We'll mention in the release notes the change in behavior which will hopefully be sufficient for existing and new users to understand this.